pkg/codegen/nodejs.go

```diff
@@ -59,6 +62,18 @@ func GenerateNodeJS(pg *PackageGenerator, name string) (map[string]*bytes.Buffer
 	}
 	files["package.json"] = bytes.ReplaceAll(packageJSON, []byte("${VERSION}"), []byte(""))
 
+	// Pin the kubernetes provider version used
+	utilities, ok := files["utilities.ts"]
+	if !ok {
+		return nil, fmt.Errorf("cannot find generated utilities.ts")
+	}
+	files["utilities.ts"] = bytes.ReplaceAll(utilities,
```

Comment:
Should we be more strict and ensure that `getVersion` already exists in the generated utilites.ts?

Reply:
I don't really see a reason to do that since the utilities.ts is the same file in every generated sdk and we have a unittest that verifies that we replace it correctly